transpose.json

```diff
@@ -0,0 +1,231 @@
+          "FOBS", 
+          "RULE", 
+          "ATOP", 
+          "CLOT", 
+          "TIME", 
+          "UNIT", 
+          "RENT", 
+          "EDGE" 
+        ]
+    },
+    {
+      "description": "triangle",
+      "input": [ 
+          "T",
+          "EE",
+          "AAA",
+          "SSSS",
+          "EEEEE",
+          "RRRRRR"
+        ],
+      "expected": [ 
+          "TEASER",
+          " EASER",
+          "  ASER",
+          "   SER",
+          "    ER",
+          "     R"
+        ]
+    },
+    {
+      "description": "many lines",
+      "input": [ 
+          "Chor. Two households, both alike in dignity,",
+          "In fair Verona, where we lay our scene,",
+          "From ancient grudge break to new mutiny,",
+          "Where civil blood makes civil hands unclean.",
+          "From forth the fatal loins of these two foes",
+          "A pair of star-cross'd lovers take their life;",
+          "Whose misadventur'd piteous overthrows",
+          "Doth with their death bury their parents' strife.",
+          "The fearful passage of their death-mark'd love,",
+          "And the continuance of their parents' rage,",
+          "Which, but their children's end, naught could remove,",
+          "Is now the two hours' traffic of our stage;",
+          "The which if you with patient ears attend,",
+          "What here shall miss, our toil shall strive to mend."
+        ],
+      "expected": [ 
+          "CIFWFAWDTAWITW",
+          "hnrhr hohnhshh",
+          "o oeopotedi ea",
+          "rfmrmash  cn t",
+          ".a e ie fthow ",
+          " ia fr weh,whh",
+          "Trnco miae  ie",
+          "w ciroitr btcr",
+          "oVivtfshfcuhhe",
+          " eeih a uote  ",
+          "hrnl sdtln  is",
+          "oot ttvh tttfh",
+          "un bhaeepihw a",
+          "saglernianeoyl",
+          "e,ro -trsui ol",
+          "h uofcu sarhu ",
+          "owddarrdan o m",
+          "lhg to'egccuwi",
+          "deemasdaeehris",
+          "sr als t  ists",
+          ",ebk 'phool'h,",
+          "  reldi ffd   ",
+          "bweso tb  rtpo",
+          "oea ileutterau",
+          "t kcnoorhhnatr",
+          "hl isvuyee'fi ",
+          " atv es iisfet",
+          "ayoior trr ino",
+          "l  lfsoh  ecti",
+          "ion   vedpn  l",
+          "kuehtteieadoe ",
+          "erwaharrar,fas",
+          "   nekt te  rh",
+          "ismdsehphnnosa",
+          "ncuse ra-tau l",
+          " et  tormsural",
+          "dniuthwea'g t ",
+          "iennwesnr hsts",
+          "g,ycoi tkrttet",
+          "n ,l r s'a anr",
+          "i  ef  'dgcgdi",
+          "t  aol   eoe,v",
+          "y  nei sl,u; e",
+          ",  .sf to l   ",
+          "     e rv d  t",
+          "     ; ie    o",
+          "       f, r   ",
+          "       e  e  m",
+          "       .  m  e",
+          "          o  n",
+          "          v  d",
+          "          e  .",
+          "          ,  "
```

Comment:
I think I missed a small thing - no spaces after the comma, right?

(I also haven't checked with code, so I may have missed others, but I  least see this)